types/applications/application.d.ts

```diff
@@ -251,7 +251,7 @@ declare class Application {
    * @param html  - The HTML element which should be filtered
    *                (unused)
    */
-  protected _onSearchFilter(event: KeyboardEvent, query: RegExp, html: HTMLElement): void;
+  protected _onSearchFilter(event: any, query: any, html: HTMLElement): void;
```

Comment:
I made this open here because it was not used. But I did not look in the other places this was used, so if it is the same in all sub classes, then I will remove that change.